alerts/auth0_bruteforce_user.conf

```diff
@@ -1,4 +1,5 @@
 [options]
 threshold_count = 1
 search_depth_min = 60
-severity = 'WARNING'
\ No newline at end of file
+severity = 'WARNING'
+email_patterns = *@mozilla.com,*@mozilla.org*@getpocket.com
```

Comment:
shouldn't it be a string:
`
email_patterns = '*@mozilla.com,*@mozilla.org,*@getpocket.com'
`